msg/templates/uorb/microRTPS_client.cpp.template

```diff
@@ -207,7 +206,7 @@ void* send(void* /*unused*/)
     struct timespec end;
     px4_clock_gettime(CLOCK_REALTIME, &end);
     double elapsed_secs = double(end.tv_sec - begin.tv_sec) + double(end.tv_nsec - begin.tv_nsec)/double(1000000000);
-    printf("\nSENT:     %lu messages in %d LOOPS, %lu bytes in %.03f seconds - %.02fKB/s\n",
+    printf("\nSENT:     " PRIu64 " messages in %d LOOPS, " PRIu64 " bytes in %.03f seconds - %.02fKB/s\n",
             sent, loop, total_sent, elapsed_secs, (double)total_sent/(1000*elapsed_secs));
```

Comment:
cinttypes actually defines __STDC_FORMAT_MACROS, but PRIu64 doesn't work without the leading "%"